aspnetcore/blazor/call-javascript-from-dotnet.md

```diff
@@ -579,42 +582,46 @@ As a more detailed example, consider the following component that renders an int
 
 The corresponding JavaScript module, which should be placed at `wwwroot/mapComponent.js`, is as follows:
 
-```js
+```javascript
 import 'https://api.mapbox.com/mapbox-gl-js/v1.12.0/mapbox-gl.js';
 
-// TO MAKE THE MAP APPEAR YOU MUST ADD YOUR ACCESS TOKEN FROM https://account.mapbox.com
-mapboxgl.accessToken = '<your access token here>';
+mapboxgl.accessToken = '{ACCESS TOKEN}';
 
 export function addMapToElement(element) {
-    return new mapboxgl.Map({
-        container: element,
-        style: 'mapbox://styles/mapbox/streets-v11',
-        center: [-74.5, 40],
-        zoom: 9
-    });
+  return new mapboxgl.Map({
+    container: element,
+    style: 'mapbox://styles/mapbox/streets-v11',
+    center: [-74.5, 40],
+    zoom: 9
+  });
 }
 
 export function setMapCenter(map, latitude, longitude) {
-    map.setCenter([longitude, latitude]);
+  map.setCenter([longitude, latitude]);
 }
 ```
 
-Finally, to produce correct styling, you should also add the following stylesheet tag to the host HTML page (`index.html` or `_Host.cshtml`):
+The placeholder `{ACCESS TOKEN}` in the preceding code is a valid access token from https://account.mapbox.com.
```

Comment:
Not sure about this phrasing! It sounds to me like "You can use the string {ACCESS TOKEN} as an access token", but you can't. Could we emphasise something like "You need to replace the string {ACCESS TOKEN} with a valid access token that you can get from ..."